CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 #### Improvements in the language
 - Implement assignable anonymous struct (fix #23855) (#23857)
 - Remove ancient deprecations (#23479)
+- Add `alignof` builtin to get type alignment.
```

Comment:
Add a new section on top, or remove the entry here (it will be mostly generated by a script the next time @medvednikov bumps the version).

The entries here, are for the already done version 0.4.10, that was released in March.